🐛(back) ignore domain check if http referer is missing (#569)
openfun/marsha: CHANGELOG.md

- Fix errors in lambda-migrate function. The timed text tracks migrations can
  lead to an infinite loop.

> - Allow to make a LTI request without HTTP referer.

Reviewer: ```suggestion
- Authorize LTI requests that have an empty HTTP referer.
```
